### PR TITLE
GET requests crashes after previous POST request

### PR DESCRIPTION
**Pull request by turing228**
It crashes at request of a public endpoint after request of an authenticated endpoint.
Because of using the same curl during DoGETrequest and DoPOSTrequest, but the last injects trash (for DoGETrequest) into curl.

### CODE REVIEW

**Comment by turing228** [comment]
Resetting curl handler before each GET or POST request thus effectively cleaning all curl options.

File: BitfinexAPI.cpp [hunk 1]
```diff
@@ -1048,15 +1051,17 @@ DoPOSTrequest(const string &UrlEndPoint, const string &params, string &result)
         httpHeaders = curl_slist_append(httpHeaders, ("X-BFX-PAYLOAD:" + payload).c_str());
         httpHeaders = curl_slist_append(httpHeaders, ("X-BFX-SIGNATURE:" + signature).c_str());
         
+		curl = curl_easy_init();
```
Comment (MMquant) [inline BitfinexAPI.cpp]: Fix indentation

File: BitfinexAPI.cpp [hunk 2]
```diff
@@ -1007,9 +1007,12 @@ DoGETrequest(const string &UrlEndPoint, const string &params, string &result)
     if(curl) {
         
         string url = APIurl + UrlEndPoint + params;
+		
+		curl = curl_easy_init();
```
Comment (MMquant) [inline BitfinexAPI.cpp]: Fix indentation

**Review by MMquant — CHANGES_REQUESTED** [review]
Fix indentation please so that I can squash and merge the patch.

**Pull request merged**